feat(wallet)_: additional properties added to the router's path for the needs of the custom tx settings part

saledjenic opened the pull request:
The following properties are added to the `routes.Path`:
- `SuggestedTxNonce`
- `SuggestedTxGasAmount`
- `SuggestedApprovalTxNonce`
- `SuggestedApprovalGasAmount`
- `TxGasFeeMode`
- `ApprovalGasFeeMode`

saledjenic committed:
feat(wallet)_: additional properties added to the router's path for the needs of the custom tx settings part

The following properties are added to the `routes.Path`:
- `SuggestedTxNonce`
- `SuggestedTxGasAmount`
- `SuggestedApprovalTxNonce`
- `SuggestedApprovalGasAmount`
- `TxGasFeeMode`
- `ApprovalGasFeeMode`

status-im-auto commented:
### Jenkins Builds
<details>
<summary>Click to see older builds (13)</summary>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 4add2623 | [#1](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6339/1/) | 2025-02-13 09:41:04 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250213-093722-4add26-pr6339.zip) |
| :heavy_check_mark: | 4add2623 | [#1](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6339/1/) | 2025-02-13 09:41:51 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250213-093721-4add26-pr6339.zip) |
| :heavy_check_mark: | 4add2623 | [#1](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6339/1/) | 2025-02-13 09:42:56 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250213-093724-4add26-pr6339.zip) |
| :heavy_check_mark: | 4add2623 | [#1](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6339/1/) | 2025-02-13 09:43:49 | ~6 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250213-093722-4add26-pr6339.zip) |
| :heavy_check_mark: | 4add2623 | [#1](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6339/1/) | 2025-02-13 09:44:08 | ~6 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250213-093718-4add26-pr6339.aar) |
| | | | | | | |
| :heavy_check_mark: | bbb23583 | [#2](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6339/2/) | 2025-02-13 09:43:16 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250213-093758-bbb235-pr6339.zip) |
| :heavy_check_mark: | bbb23583 | [#2](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6339/2/) | 2025-02-13 09:45:18 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250213-094114-bbb235-pr6339.zip) |
| :heavy_check_mark: | bbb23583 | [#2](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6339/2/) | 2025-02-13 09:46:42 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250213-094155-bbb235-pr6339.zip) |
| :heavy_check_mark: | bbb23583 | [#2](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6339/2/) | 2025-02-13 09:48:20 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250213-094301-bbb235-pr6339.zip) |
| :heavy_check_mark: | bbb23583 | [#2](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6339/2/) | 2025-02-13 09:49:48 | ~11 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6339/2//consoleText) |
| :heavy_check_mark: | bbb23583 | [#2](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6339/2/) | 2025-02-13 09:55:19 | ~11 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250213-094353-bbb235-pr6339.zip) |
| :heavy_check_mark: | bbb23583 | [#2](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6339/2/) | 2025-02-13 09:56:35 | ~12 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250213-094413-bbb235-pr6339.aar) |
| :heavy_check_mark: | bbb23583 | [#2](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6339/2/) | 2025-02-13 10:08:16 | ~30 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6339/2//consoleText) |
</details>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 24e05cfc | [#3](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6339/3/) | 2025-02-13 13:08:42 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250213-130455-24e05c-pr6339.zip) |
| :heavy_check_mark: | 24e05cfc | [#3](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6339/3/) | 2025-02-13 13:08:50 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250213-130451-24e05c-pr6339.zip) |
| :heavy_check_mark: | 24e05cfc | [#3](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6339/3/) | 2025-02-13 13:08:51 | ~3 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250213-130459-24e05c-pr6339.zip) |
| :heavy_check_mark: | 24e05cfc | [#3](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6339/3/) | 2025-02-13 13:10:05 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250213-130453-24e05c-pr6339.zip) |
| :heavy_check_mark: | 24e05cfc | [#3](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6339/3/) | 2025-02-13 13:10:14 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250213-130455-24e05c-pr6339.zip) |
| :heavy_check_mark: | 24e05cfc | [#3](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6339/3/) | 2025-02-13 13:12:19 | ~7 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250213-130450-24e05c-pr6339.aar) |
| :heavy_check_mark: | 24e05cfc | [#3](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6339/3/) | 2025-02-13 13:16:23 | ~11 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6339/3//consoleText) |
| :heavy_multiplication_x: | 24e05cfc | [#3](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6339/3/) | 2025-02-13 13:36:21 | ~31 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6339/3//consoleText) |
| | | | | | | |
| :heavy_check_mark: | d2c220fe | [#4](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6339/4/) | 2025-02-13 14:24:39 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250213-142011-d2c220-pr6339.zip) |
| :heavy_check_mark: | d2c220fe | [#4](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6339/4/) | 2025-02-13 14:25:33 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250213-142014-d2c220-pr6339.zip) |
| :heavy_check_mark: | d2c220fe | [#4](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6339/4/) | 2025-02-13 14:25:34 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250213-142012-d2c220-pr6339.zip) |
| :heavy_check_mark: | d2c220fe | [#4](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6339/4/) | 2025-02-13 14:25:35 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250213-142018-d2c220-pr6339.zip) |
| :heavy_check_mark: | d2c220fe | [#4](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6339/4/) | 2025-02-13 14:26:05 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250213-142012-d2c220-pr6339.zip) |
| :heavy_check_mark: | d2c220fe | [#4](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6339/4/) | 2025-02-13 14:26:36 | ~6 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250213-142009-d2c220-pr6339.aar) |
| :heavy_multiplication_x: | d2c220fe | [#4](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6339/4/) | 2025-02-13 14:37:14 | ~16 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6339/4//consoleText) |
| :heavy_check_mark: | d2c220fe | [#4](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6339/4/) | 2025-02-13 14:55:34 | ~35 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6339/4//consoleText) |
| :heavy_multiplication_x: | d2c220fe | [#5](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6339/5/) | 2025-02-13 15:05:56 | ~16 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6339/5//consoleText) |
| :heavy_multiplication_x: | d2c220fe | [#6](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6339/6/) | 2025-02-13 19:08:15 | ~9 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6339/6//consoleText) |
| :heavy_check_mark: | d2c220fe | [#7](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6339/7/) | 2025-02-13 19:33:06 | ~12 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6339/7//consoleText) |

saledjenic committed:
fix(wallet)_: applied custom tx changes to the path

Before this change, custom parameters were applied only after the next fee update, now they will
be updated appropriately and immediately.
